### DISCUSSION TITLE
Unable to compile Rust hello world on Windows: linker link.exe not found

### DISCUSSION THREAD

**Discussion by strong-728** [community/community#35134] [Other Feature Feedback, Questions, & Ideas]
I have installed Rust on windows from [Rust installation page](https://www.rust-lang.org/tools/install). After installation I tried running the "hello world" program but got the following error.

error: linker `link.exe` not found
note: The system cannot find the file specified. (os error 2)
note: the msvc targets depend on the msvc linker but `link.exe` was not found
note: please ensure that VS 2013, VS 2015 or VS 2017 was installed with the Visual C++ option
error: aborting due to previous error
error: Could not compile `helloworld`.

**Comment by sloperun3az** [comment]
I enjoy playing video games immensely. I can only unwind after a day of labor in this manner. I wish to immerse myself in the game and appreciate its enchanted universe. Therefore, many thanks for the game **<a href="https://poppyplaygame.io/">poppy playtime</a>**. You suggested.

**Comment by sohamdascs** [comment]
do these in terminal
1. rustup uninstall toolchain stable-x86_64-pc-windows-msvc
2. rustup toolchain install stable-x86_64-pc-windows-gnu 
3. rustup default stable-x86_64-pc-windows-gnu

**Reply by SonicWim** [reply]
Perhaps JetBrains should add this info to their Rust toolchain setup instructions because they say "Just accept the defaults", but this installs the MSVC via Visual Studio. That one should also work (and comes in 4 version on my PC) and as a batfile to set things up but, just using the GNU version saves you the hassle...

**Comment by goUp9** [comment]
Thanks @sohamdascs 
It works.
